src/main/java/com/notably/logic/suggestion/parser/OpenSuggestionCommandParser.java

```diff
@@ -0,0 +1,54 @@
+package com.notably.logic.suggestion.parser;
+
+import static com.notably.logic.parser.CliSyntax.PREFIX_TITLE;
+
+import java.util.Optional;
+
+import com.notably.commons.core.path.AbsolutePath;
+import com.notably.logic.correction.AbsolutePathCorrectionEngine;
+import com.notably.logic.parser.ArgumentMultimap;
+import com.notably.logic.parser.ArgumentTokenizer;
+import com.notably.logic.parser.ParserUtil;
+import com.notably.logic.parser.exceptions.ParseException;
```

Comment:
Leaving this here for future reference. Don't need to change anything for now.

I think the current directory structure of separating `com.notably.logic.suggestion` from `com.notably.logic.parser` + `com.notably.logic.command` leads to awkward importing. For example, over here, we're importing utility classes from the `com.notably.parser` package, even though this file is located under `com.notably.logic.suggestion.parser`.

Perhaps, the more natural way of organizing would be:
- `/com/notably/parser`: generic parsers (called when a user presses <Enter>)
- `/com/notably/parser/suggestion`: suggestion parsers (called every time input changes)
- `/com/notably/command`: commands created by `/com/notably/parser`
- `/com/notably/command/suggestion`: commands created by `/com/notably/parser/suggestion`

This way, we can import leave all the common utilities (such as `ArgumentMultimap`, for example) in either `/com/notably/parser` or `/com/notably/command`, and either `/com/notably/parser/suggestion` or `/com/notably/command/suggestion` can use them more naturally.

@johannagwan what do you think?

Reply:
Alrighties suresies!! Sounds good.

Reply:
I shall create another PR for the reorganization.